docs/theming-flatpak.rst

```diff
@@ -0,0 +1,56 @@
+
+Installing themes
+-----------------
+
+The current themes are packaged in the flathub repository which you can add (if it's not already added) by running::
+
+  $ flatpak remote-add flathub https://flathub.org/repo/flathub.flatpakrepo
+
+To see a list of currently packaged themes you can run::
+
+  $ flatpak remote-ls flathub | grep org.gtk.Gtk3theme
+
+The list as of posting is:
+
+- org.gtk.Gtk3theme.Adapta
+- org.gtk.Gtk3theme.Adapta-Eta
+- org.gtk.Gtk3theme.Adapta-Nokto
+- org.gtk.Gtk3theme.Adapta-Nokto-Eta
+- org.gtk.Gtk3theme.Ambiance
+- org.gtk.Gtk3theme.Arc
+- org.gtk.Gtk3theme.Arc-Dark
+- org.gtk.Gtk3theme.Arc-Dark-solid
+- org.gtk.Gtk3theme.Arc-Darker
+- org.gtk.Gtk3theme.Arc-Darker-solid
+- org.gtk.Gtk3theme.Arc-solid
+- org.gtk.Gtk3theme.Breeze
+- org.gtk.Gtk3theme.Breeze-Dark
+- org.gtk.Gtk3theme.Greybird
+- org.gtk.Gtk3theme.Numix
+- org.gtk.Gtk3theme.Pop
+- org.gtk.Gtk3theme.Pop-light
+- org.gtk.Gtk3theme.elementary
+
+You can install them with ``flatpak install flathub org.gtk.Gtk3theme.Foo``.
+
+Applying themes
+---------------
+
+There is no way to specify the theme Flatpak applications use. The applications will try to match the system theme currently being used, if it corresponds to any of the Flatpak themes installed, and will fall back to Adwaita if a corresponding theme isn't detected. This happens because the ``flatpak`` tool doesn’t depend upon Gtk so it will not automatically detect your theme.
+
+The theming system requires Flatpak 0.8.4+ and applications using up to date ``org.gnome.Platform`` 3.24+.
+
+Other notes on theming
+----------------------
+
+The same solution exists for icon themes and more will be packaged soon. They can be searched with ``flatpak remote-ls flathub grep org.freedesktop.Platform.Icontheme``.
+
+This process will likely be automated by Gnome-Software at some point.
+
+Themes are picked up via GtkSettings, which means you require a settings-daemon running to set this. On GNOME (and GNOME-derived desktop environments) this will already be running but for other desktops you can run ``gsd-xsettings`` directly in your libexec directory provided by ``gnome-settings-daemon``.
+
+If you use the *Global Dark Theme* option in ``gnome-tweak-tool`` it will not work as that simply writes to ``settings.ini`` which isn’t available in the sandbox. Use dark versions of the theme instead if they exist.
```

Comment:
GNOME-Tweaks removes this option in 3.28 which might be worth noting. Also perhaps better describe how to "use dark versions of themes".